src/main/java/salesforce/ui/pages/product/NewProductPage.java

```diff
@@ -0,0 +1,112 @@
+package salesforce.ui.pages.product;
```

Comment:
You should add the copyright on all the classes.

Reply:
Thanks. I have just fixed that